src/App.js

```diff
@@ -19,14 +28,20 @@ const App = () => {
     setCounterText(event.target.value)
   }
 
+  const saveCounterNameCountLocally = () =>{
+    localStorage.setItem("count",JSON.stringify(count))
```

Comment:
also instead of mentioning the string directly here export it as a constant from the storage.js file